tezos/context/src/working_tree/working_tree.rs

```diff
@@ -712,108 +712,12 @@ impl WorkingTree {
 
         let mut storage = self.index.storage.borrow_mut();
 
-        match self.index.find_node(dir_id, key, &mut storage) {
+        match self.index.go_to_node(dir_id, key, &mut storage) {
             Ok(node_id) => node_id.is_some(),
             Err(_) => false,
         }
     }
 
-    // TODO: recursion is risky (stack overflow) and inefficient, try to do it iteratively..
```

Comment:
@sebastiencs not used anymore? (I was wondering why we had what seemed to be duplicate code here and in the context)

Reply:
@tizoc Yes it's a duplicate.
I'm not sure but at some point we moved some methods from the `WorkingTree` into `TezedgeIndex`, and I think we didn't remove some from the `WorkingTree`